TackleTabby/Assets/Scripts/Combos/ComboTracker.cs

```diff
@@ -0,0 +1,44 @@
+using UnityEngine;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.Events;
+
+public class ComboTracker : MonoBehaviour
+{
+    [SerializeField]
+    private int ComboLength = 2;
+    [SerializeField]
+    private Combo _endingCombo;
```

Comment:
this doesn't follow our code conventions
```suggestion
    private Combo EndingCombo;
```